src/components/Table/Table.scss

```diff
@@ -33,6 +33,23 @@ $c-table-row-selected-bg: $color-primary !default;
 	th {
 		font-weight: 500;
 	}
+
+	tr {
+		line-height: 32px;
+	}
+
+	tr {
+		> td,
+		> th {
+			&:first-child {
+				padding-left: 5px;
```

Comment:
Deze mss enkel toevoegen als ze clickable zijn en best ook dan op `:last-child`

Reply:
> best ook dan op :last-child
done

> Deze mss enkel toevoegen als ze clickable zijn
lijkt me veel werk voor weinig return

Reply:
Veel werk 😄? Je voegt al een u-clickable class toe dus je kon gwn dit doen:
```css
tr {
	&.u-clickable {
		> td,
		> th {
			&:first-child {
				padding-left: 5px;
				....
```
Maar zo veel verschil zal het niet maken dus laat maar zo.

Reply:
ah, ja, ik dacht dat ik opnieuw ging moeten controleren of de onRowClick handler bestaat en dan een extra class toevoegen, maar u-clickable is vrij eenvoudig